tests/sentry/web/frontend/test_msteams_extension.py

```diff
@@ -19,11 +26,18 @@ def test_map_params(self):
 
     @patch("sentry.web.frontend.msteams_extension_configuration.unsign")
     def test_expired_signature(self, mock_unsign):
-        with self.feature({"organizations:integrations-msteams": True}):
+        with self.feature(
+            {
+                "organizations:integrations-msteams": True,
+                "organizations:integrations-alert-rule": True,
+            }
+        ):
             mock_unsign.side_effect = SignatureExpired()
-            self.login_as(self.user)
-            org = self.create_organization()
-            OrganizationMember.objects.create(user=self.user, organization=org)
-            path = u"/extensions/msteams/configure/"
-            resp = self.client.get(path, {"signed_params": "test"})
+            resp = self.hit_configure({"signed_params": "test"})
             assert "Installation link expired" in resp.content
+
+    def test_no_feature_flags(self):
+        with self.feature({"organizations:integrations-msteams": True}):
```

Comment:
this confused me because you have the `msteams` flag set to `True`, can you name it `test_no_team_plan_feature_flag` or something?

Reply:
(I realize that once we get rid of the msteams flag this will make more sense so not blocking)